ui/src/overlays/components/OverlayController.tsx

```diff
@@ -8,26 +8,67 @@ import {AppState} from 'src/types'
 // Components
 import {Overlay} from '@influxdata/clockface'
 import NoteEditorOverlay from 'src/dashboards/components/NoteEditorOverlay'
+import AllAccessTokenOverlay from 'src/authorizations/components/AllAccessTokenOverlay'
+import BucketsTokenOverlay from 'src/authorizations/components/BucketsTokenOverlay'
+import {dismissOverlay} from 'src/overlays/actions/overlays'
 
 interface StateProps {
   overlayID: string
```

Comment:
Can we type overlayID as `'add-master-token' | 'add-token' | ....` instead of string?

Reply:
on it!